lib/queue_classic/worker.rb

```diff
@@ -161,5 +163,15 @@ def setup_queues(adapter, queue, queues, top_bound)
       end
     end
 
+    def clone_args(job)
+      job[:args].map do |arg|
+        begin
+          arg.clone
+        rescue TypeError
```

Comment:
Why the rescue? What use case is it covering? I don't see anything in the tests covering that.